diagnostics/TC_Rain/doc/TC_Rain.rst

```diff
@@ -0,0 +1,107 @@
+   of reading. Multi-line comments or commands like this need to be indented by
+   exactly three spaces.
+
+.. Underline with '='s to set top-level heading: 
+   https://docutils.sourceforge.io/docs/user/rst/quickref.html#section-structure
+
+TC Rain Rate Azimuthal Average Documentation
+================================
+
+Last update: 5/27/2022
+
+This POD calculates and plots azimuthal averages for tropical cyclone (TC) rain rates 
+from TC track data and model output precipitation. 
+
+
+.. Underline with '-'s to make a second-level heading.
+
+Version & Contact info
+----------------------
+
+- Version/revision information: version 1 (5/06/2020)
+- PI Daehyun Kim, University of Washington, daehyun@uw.edu
+- Developer/point of contact Nelly Emlaw, University of Washington, gnemlaw@uw.edu
+
+
+Open source copyright agreement
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The MDTF framework is distributed under the LGPLv3 license (see LICENSE.txt). 
+Unless you've distributed your script elsewhere, you don't need to change this.
+
+Functionality
+-------------
+
+In the current draft code, the data is loaded in first. The netcdf file for the total 
+precipitation is loaded in through xarray. The TC track data is read in line by line from a 
+txt file. 
+
+Then the azimuthal average is calulated for a set of discrete radii by measuring the 
+distance between a data point and the center of the storm for each snapshot. 
+
+A plot showing TC rain rate as a function of radius is made for select snapshopts that 
+reach 35-45 knot max wind speeds.
+
+Required programming language and libraries
+-------------------------------------------
+
+matplotlib, numpy, netcdf, xarray, scipy
+
+Required model output variables
+-------------------------------
+
+Total Precipitation 
+
+TC Track Data (required: storm center latitude and longitufe, time for each snapshot,
+optional: max windspeed, minimum central surface pressure, sst, etc)
+
+References
+----------
+
+1. Kim, D., Y. Moon, S. Camargo, A. Sobel, A. Wing, H. Murakami, G. Vecchi, M. Zhao, 
+and E. Page, 2018: Process-oriented diagnosis of tropical cyclones in high-resolution
+climate models. J. Climate, 31, 1685–1702, https://doi.org/10.1175/JCLI-D-17-0269.1.
+
+2. Moon, Y., D. Kim, S. Camargo, A. Wing, A. Sobel, H. Murakami, K. Reed, G. Vecchi, 
+M. Wehner, C. Zarzycki, and M. Zhao, 2020: Azimuthally averaged wind and 
+thermodynamic structures of tropical cyclones in global climate models and their 
+sensitivity to horizontal resolution. J. Climate, 33, 1575–1595, 
+https://doi.org/10.1175/JCLI-D-19-0172.1
+
+3. Moon, Y., D. Kim, A. A. Wing, S. J. Camargo, M. Zhao, L. R. Leung, M. J. Roberts, 
+D.-H. Cha, and J. Moon: An evaluation of global climate model-simulated tropical 
+cyclone rainfall structures in the HighResMIP against the satellite observations, J.
+Climate, Accepted.
+
+
+More about this diagnostic
+--------------------------
```

Comment:
Would it be possible to add some more text about how a model developer could use this diagnostic?  What should one be looking for in terms of the radial distribution of precip.?  Are there common biases to be on the lookout for?   What would a ideal distribution look like?